prov/efa/src/rxr/rxr_ep.c

```diff
@@ -642,7 +651,7 @@ static void rxr_ep_free_res(struct rxr_ep *rxr_ep)
 		tx_entry = container_of(entry, struct rxr_tx_entry,
 					queued_rnr_entry);
 		FI_WARN(&rxr_prov, FI_LOG_EP_CTRL,
-			"Closing ep with queued tx_entry: %p\n",
+			"Closing ep with queued rnr tx_entry: %p\n",
```

Comment:
This should be in the first commit for tx, not here

Reply:
fixed in new revision. Thank you!